pyquil/setup/pyquil_config_setup.py

```diff
@@ -24,10 +27,47 @@ def main():
     print("Enter the required information below for Forest connections.")
     print("If you haven't signed up yet you will need to do so first at http://forest.rigetti.com")
 
-    key = input("Forest API Key: ")
-    user = input("User ID: ")
+    while True:
+        key = input("Forest API Key: ")
+        key_ma = re.match(r'^\s*(\w+)\s*$', key)
+        if key_ma:
+            key = key_ma.group(1)
+            if len(key) == 40:
```

Comment:
Might as well just change the regex to \w{40} and do this in one go

Reply:
done